core/shared/src/main/scala/scalaz/zio/internal/FiberContext.scala

```diff
@@ -49,6 +54,74 @@ private[zio] final class FiberContext[E, A](
   private[this] val locked          = Stack[Executor]()
   private[this] val supervised      = Stack[Set[FiberContext[_, _]]]()
 
+  private[this] val tracingStatus = if (tracingEnabled) StackBool(initialTracingStatus) else null
```

Comment:
Could you add a blank line in-between those? Otherwise a bit hard to read.

Reply:
done!